### PR TITLE
translator: remove IssuingValidator #352

### PR DESCRIPTION
**Pull request by adecaro**
Signed-off-by: Angelo De Caro <adc@zurich.ibm.com>

### CODE REVIEW

File: token/services/vault/translator/translator_test.go [hunk 1]
```diff
@@ -38,7 +38,7 @@ var _ = Describe("Translator", func() {
 		fakeIssuingValidator = &mock.IssuingValidator{}
```
[the remark this answers is not in the document]
Comment (adecaro) [reply token/services/vault/translator/translator_test.go:38]: yes, good catch.

**Comment by adecaro** [comment]
Thanks @mbrandenburger , comment addressed.

**Comment by mbrandenburger** [comment]
> comment addressed.

I guess you missed one :D

> While the changes look good to me, it is unclear why this is removed. Can you please add some explanation why this is removed.

**Comment by adecaro** [comment]
Sorry, @mbrandenburger . I added it to the git issue. Essentially, the validator is responsible of checking that the identity of an issuer is valid 😅

**Review by mbrandenburger — APPROVED** [review]

**Pull request merged**